Header header

float32 gim_x
float32 gim_y
float32 rad_x
float32 rad_y
float32 dis
int32 info_type